packages/web-components/fast-foundation/src/tree-item/tree-item.template.ts

```diff
@@ -26,31 +26,29 @@ export function treeItemTemplate<T extends FASTTreeItem>(
                 filter: elements(),
             })}
         >
-            <div class="positioning-region" part="positioning-region">
-                <div class="content-region" part="content-region">
-                    ${when(
-                        x => x.childItems && x.childItemLength > 0,
-                        html<T>`
-                            <div
-                                aria-hidden="true"
-                                class="expand-collapse-button"
-                                part="expand-collapse-button"
-                                @click="${(x, c) =>
-                                    x.handleExpandCollapseButtonClick(
-                                        c.event as MouseEvent
-                                    )}"
-                                ${ref("expandCollapseButton")}
-                            >
-                                <slot name="expand-collapse-glyph">
-                                    ${options.expandCollapseGlyph ?? ""}
-                                </slot>
-                            </div>
-                        `
-                    )}
-                    ${startSlotTemplate(options)}
+            <div class="control" part="control">
```

Comment:
Went looking to investigate the test failure and noticed the use of control here. I thought we discussed but this is confusing as the _control_ is the host element in cases like these. Similar to radiogroup, which is also the control but has an element which enables the positioning of elements to one another (such as the label to the radios in that case). Did we end up documenting what "control" means if it's going to be used universally in scenarios where it doesn't necessarily represent the entirety of the control (in this case, it's a portion)?

Reply:
The notes I have regarding "control" was that it represented either the underlying element being wrapped, as in the case of Button or Anchor, or the logical representation of the control you interact with. In the latter case, that's something like the square portion of a Checkbox (but not the label), or the button for a Select or Combobox (but not the listbox).

We extended this to components including Dialog, Toolbar, and Tooltip as there are often other elements that might be required (modal overlay, label, and anchored region respectively).

In the case of Tree Item the "control" is the single row for the item you interact with (that gets hover treatment, etc) while the "other elements" are the child items (which are not logically thought of as part of that control, even though they are contained within).

Regarding the history on this, at one point we had explicit names like "dialog", "toolbar", etc. but we agreed consistency between the components was helpful in this case, and that "control" was a reasonable representation of the logical footprint of what you think of styling when you think of that type of component.

We also considered "container", but the lines between accurate usage of "control" and "container" seemed ambiguous in some cases. We leaned toward fewer names as a way to encourage familiarity between the components.

@EisenbergEffect in case you recall more of the conversation I didn't write down.